scripts/sync-down-from-production.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env sh
+set -xe
+
+SCRIPT_DIR=$(dirname "$0")
+
+# Load up environment variables for the local (docker) postgres
+. "$SCRIPT_DIR"/../.env
+
+# Check wheter `docker compose` or `docker-compose` is available
+if command -v docker-compose > /dev/null; then
+  DOCKER_COMPOSE=docker-compose
+elif command -v docker compose > /dev/null; then
+  DOCKER_COMPOSE="docker compose"
+else
+  echo "Neither docker compose nor docker-compose is available"
+  exit 1
+fi
```

Comment:
Trying to make it as flexible as possible, so we can run this on dev env, pizzas, and staging